app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposerInput.kt

```diff
@@ -202,9 +210,17 @@ private fun InputContent(
         MessageComposerTextInput(
             isTextExpanded = isTextExpanded,
             inputFocused = inputFocused,
-            colors = inputType.inputTextColor(),
+            colors = viewModel.state().duration?.let {
+                wireTextFieldColors(
+                    backgroundColor = Color.Transparent,
+                    borderColor = Color.Transparent,
+                    focusColor = Color.Transparent,
+                    placeholderColor = colorsScheme().primary
+                )
+            } ?: inputType.inputTextColor(),
```

Comment:
suggestion: move that to dedicated fun like `inputType.selfDeletingInputTextColor()` or just reuse `inputTextColor()` like:
```
@Composable
fun inputTextColor(isSelfDeleting: Boolean = false): WireTextFieldColors = wireTextFieldColors(
        backgroundColor = Color.Transparent,
        borderColor = Color.Transparent,
        focusColor = Color.Transparent,
        placeholderColor = if (isSelfDeleting) colorsScheme().primary else colorsScheme().secondaryText
    )
```